src/reducers/schemas/selectors.js

```diff
@@ -1,42 +1,140 @@
 import { createSelector } from 'reselect'
 
-// Groups schemas into groups
-// Ie. All Data Element related into 'dataElement'
-function schemaGroups(schemas) {
-    const groups = {}
-
-    schemas.forEach(s => {
-        if (!groups[s.group]) {
-            groups[s.group] = []
+/**
+ * Helpers
+ * =======
+ * =======
+ * =======
+ */
+
+/**
+ * Group and sort schemas
+ * ======================
+ */
+const groupSchemas = schemas =>
+    schemas.reduce((groups, schema) => {
+        const { group } = schema
+
+        return {
+            ...groups,
+            [group]: groups[group] ? [...groups[group], schema] : [schema],
```

Comment:
This could use a comment, or be separated into multiple lines to simplify reading it.

Reply:
done